### PR TITLE
borrow & supply for each asset

### PR DESCRIPTION
**Pull request by dmytro-shash**
add supply & borrow for each asset, covered with test

### CODE REVIEW

File: contracts/controller/src/views.rs [hunk 1]
```diff
@@ -50,12 +52,14 @@ impl Contract {
                 let total_borrows = self.get_total_borrows(user_id.clone());
                 let total_supplies = self.get_total_supplies(user_id.clone());
                 let health_factor = self.get_health_factor(user_id.clone());
+                let user_profile = self.user_profiles.get(user_id).unwrap();
```
Comment (pahomas) [inline contracts/controller/src/views.rs:55]: what if user_profile won't be found? If you implemented a Default state, so you can use unwrap_or_default

unwrap could cause a panic effect, not sure that it good manner

File: contracts/dtoken/tests/sim/withdraw.rs [hunk 2]
```diff
@@ -392,3 +394,62 @@ fn scenario_withdraw_error_transfer() {
         view_balance(&controller, Supply, user.account_id(), dtoken.account_id());
     assert_eq!(user_balance, 20, "Balance should be 20");
 }
+
+#[test]
+fn scenario_view_accounts() {
+    // TODO remove in future if we will make sure it works properly
+    let (dtoken, controller, utoken, user) = supply_borrow_withdraw_fixture();
+
```
Comment (pahomas) [inline contracts/dtoken/tests/sim/withdraw.rs:402]: You should also check when the user profile doesn't exists flow

**Review by pahomas — CHANGES_REQUESTED** [review]

File: contracts/controller/src/views.rs (hunk 1 above)
Comment (dmytro-shash) [reply contracts/controller/src/views.rs:55]: we have the check above that account has to be present in the user_profiles `filter(|user_id| self.user_profiles.get(user_id).is_some())` so i suppose there are no case that we will be trying to get account that is not exist

File: contracts/dtoken/tests/sim/withdraw.rs (hunk 2 above)
Comment (dmytro-shash) [reply contracts/dtoken/tests/sim/withdraw.rs:402]: i passed the account bob.near which is not supplied or borrowed any so there no response for him

**Review by pahomas — APPROVED** [review]

**Pull request merged**